styles.css

```diff
@@ -92,5 +97,170 @@ body {
+}
+
+.recent-title {
+  display: inline-block;
+  border-bottom: 1px solid black;
+  margin: 0 20px 0 20px;
+  height: 83px;
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+  margin-bottom: 70px;
+}
+
+.recent-title2 {
+  display: inline-block;
+  margin: 0 20px 0 20px;
+  height: 36px;
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+}
+
+.contend {
+  font-size: 16px;
+  padding: 0 24px 0 24px;
+  width: 327px;
+}
+
+.my-recent ul {
+  display: flex;
+  width: auto;
+  justify-content: space-evenly;
+  list-style-type: none;
+  padding: 0;
+}
+
+.my-recent li {
+  font-size: 20px;
+  font-weight: 500;
+  background-color: transparent;
+  padding: 10px 12px;
+  border: 1px solid  #8993a4;
+}
+
+.see-style {
+  background-color: #ff6b00;
+  text-decoration: none;
+  height: 24px;
+  width: 100px;
+  padding: 7px 24px 6px 10px;
+  color: #fff;
+  font-family: 'Roboto', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+}
+
+.bak-image {
+  display: flex;
+  flex-direction: column;
+  background-image: url("pictures/Img\ Placeholder\ \(1\).svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.works {
+  color: #fff;
+}
+
+.works-title {
+  width: 295px;
+  font-size: 32px;
+  font-weight: 400;
+  margin: 100px 16px 0 16px;
+  line-height: 44px;
+}
+
+.works-contend {
+  width: 295px;
+  font-size: 16px;
+  margin: 0 16px 0 16px;
+}
+
+.works ul {
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  list-style-type: none;
+  padding: 40px 80px 10px 40px;
+}
+
+.works li {
+  font-size: 20px;
+  font-weight: 500;
+  background-color: rgba(197, 192, 192, 0.3);
+  padding: 10px 12px;
+  border: 1px solid  #8993a4;
+}
+
+.proyect-button {
+  margin: 30px px 0 0;
+  display: inline-block;
+  text-align: center;
+  background-color: #ff6b00;
+  text-decoration: none;
+  height: 24px;
+  padding: 9px 24px 7px 10px;
+  color: #fff;
+  font-family: 'Roboto', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+}
+
+.about-title {
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  color: #ff6b00;
+  font-size: 72px;
+  font-weight: 400;
+}
+
+.about-contend {
+  font-family: 'Crete Round', 'Open Sans', 'Helvetica Neue', sans-serif;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.about-me a {
+  margin: 30px px 0 0;
+  display: inline-block;
+  text-align: center;
+  background-color: #ff6b00;
+  text-decoration: none;
+  height: 24px;
+  padding: 9px 24px 7px 10px;
+  color: #fff;
+  font-family: 'Roboto', 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+}
+
+.ornament2 {
+  background-image: url("/Porfolio/pictures/Group2.svg");
+  background-size: cover;
+  background-position: left;
+  width: 170px;
+  height: 270px;
+  padding: 100px 0 0 0;
```

Comment:
- [ ]  You should use the background image on your about section similar to the one provided from template. I have tried to red-border your background color, and I found that its not right compare to the one provide from figma design. Please fix it to make you website looks more professional. 

Your design 
![image](https://user-images.githubusercontent.com/43877689/200125615-46984cbe-0b2b-4be9-a08c-9334287edfa5.png)

Template design 
![image](https://user-images.githubusercontent.com/43877689/200125669-2202f565-6465-4052-8e7f-4b71c2f053d5.png)